public/apps/account/password-reset-panel.tsx

```diff
@@ -33,6 +36,7 @@ import { PASSWORD_INSTRUCTION } from '../apps-constants';
 import { constructErrorMessageAndLog } from '../error-utils';
 import { validateCurrentPassword } from '../../utils/login-utils';
 import { getDashboardsInfo } from '../../utils/dashboards-info-utils';
+import { PasswordStrengthBar } from '../configuration/utils/password-strength-bar';
```

Comment:
Should the exported component follow lowercase naming conventions?

Reply:
Since this is a component and not a function the naming convention is to use an upper case. The above examples are for helper functions, which would use camel case.